jms/src/main/kotlin/io/qalipsis/plugins/jms/consumer/JmsConsumerConverter.kt

```diff
@@ -73,12 +89,18 @@ internal class JmsConsumerConverter<O : Any?>(
         eventsLogger?.info("${eventPrefix}.received.records", 1, tags = eventTags)
         consumedRecordsCounter?.increment()
 
-        val bytesCount = if (value is TextMessage) {
-            value.text.toByteArray().size
-        } else if (value is BytesMessage) {
-            value.bodyLength.toInt()
-        } else {
-            0
+        val bytesCount = when (value) {
+            is TextMessage -> {
```

Comment:
Inline the statement when there is not much code:
```suggestion
            is TextMessage -> value.text.toByteArray().size
```